src/index.d.ts

```diff
@@ -11,7 +11,7 @@
 
 export type BciAvIdType = number | (string|number)[];
 
-export type OptionsType = {
+export type BlissCellType = {
```

Comment:
I'm not sure about "BlissCellType".  In the future the cells will contain symbols other than Bliss but they will behave the same and will have similar options.  The difference will be the `bciAvId` property which will be replaced with some other kind of reference to the actual graphic of the symbol.  How about "SymbolCellType" as a generic name? Or, perhaps "ActionCellType", since the corresponding components are `Action<something>Cell`. Or, perhaps it does need to be more specific to allow different kinds of symbols.  In that case, how about "BlissSymbolCellType".  And if we support, say, the [Widgit](https://www.widgit.com/) symbols, they would be "WidgitSymbolCellType".  What do you think?

Reply:
`BlissSymbolCellType` sounds great. Having `Bliss` in the name is because this type structure contains a `bciAvId` field that is specific to Bliss symbols. When we support the graphic symbol or other type of symbols, this field will likely be replaced by the path to the graphic or identifiers to that type of symbol.